### PR TITLE
Rename `cache` directory to `.cache`

### PR DESCRIPTION
**Pull request by MasloMaslane**
I also added code for moving `cache` to `.cache` after updating to a newer version

### CODE REVIEW

File: src/sinol_make/util.py [hunk 1]
```diff
@@ -265,6 +268,16 @@ def get_file_md5(path):
...
 
 
+def make_version_changes():
+    """
+    Function to make changes after updating sinol-make.
+    """
+    # Move `cache` directory to `.cache` directory if the version is higher than 1.5.2.
+    if compare_versions("1.5.2", sinol_make.__version__) == -1:
+        if check_if_package() and os.path.exists("cache"):
+            shutil.move(os.path.join(os.getcwd(), "cache"), os.path.join(os.getcwd(), ".cache"))
```
Comment (tonowak) [inline src/sinol_make/util.py:278]: I'm not sure I understand the `if` here. When the version is strictly higher than `1.5.2`, it moves `cache` to `.cache`?

Seeing that currently sinol-make doesn't have many users, I would say it's OK to not move and just let the user have both the `cache` and `.cache` directories, just for code simplicity. I think that people that used the old sinol-make won't use the new one on the packages that they created previously.

Comment (MasloMaslane) [reply src/sinol_make/util.py]: Sure, I deleted this part of code

**Review by tonowak — COMMENTED** [review]
Instead of having a `".cache"` constant everywhere in the code, maybe we could do better?

**Comment by tonowak** [comment]
What you think would be the cleanest solution?

**Comment by MasloMaslane** [comment]
Probably a function `get_cache_path()` - returns path to cache or `get_path_in_cache(path)` - returns `./.cache/<path>`

**Comment by tonowak** [comment]
Sounds reasonable. Maybe we could also do it for other constants, if there are any?

**Comment by MasloMaslane** [comment]
Sure, I'll look for them

**Review by tonowak — APPROVED** [review]
I would say that `get_cache_path` is a bit better name than `get_path_in_cache` (and similar to e.g. executions), as the `get_path_in_cache` can suggest that we're finding a path in some cache that we're currently maintaining.

**Comment by MasloMaslane** [comment]
`get_path_in_cache(paths)` returns a path inside of cache, it works like `os.path.join`

**Comment by tonowak** [comment]
Right, and that's what I mean. It makes it sound like in the coffee you have some cache object, and not in the file system. The function name doesn't sound like it's appending the string "cache" to the path.

**Comment by MasloMaslane** [comment]
I removed the `get_path_in_*()` functions

**Pull request merged**